tests/static/all-tests.html

```diff
@@ -23,7 +23,8 @@
             "./tests-binder-radio.html",
```

Comment:
The first 6 tests listed here fail when run directly in the browser

Reply:
Fixed, but with a caveat.  QUnit.composite itself fails in Chrome and Opera when the content is not hosted by testem (likely because of file:// URL sandboxing).  Here's the error.

```
Uncaught DOMException: Blocked a frame with origin "null" from accessing a cross-origin frame.
    at HTMLIFrameElement.onIframeLoad (file:///Users/duhrer/Source/rtf/gpii-binder/node_modules/infusion/tests/lib/qunit/addons/composite/qunit-composite.js:59:13)
```

Reply:
That's ok, we're used to this restriction from other projects and you explain the process for hosting in your README